internal/host/validator.go

```diff
@@ -244,61 +251,86 @@ type validator struct {
 }
 
 func (v *validator) isConnected(c *validationContext) ValidationStatus {
-	return boolValue(c.host.CheckedInAt.String() == "" || time.Since(time.Time(c.host.CheckedInAt)) <= MaxHostDisconnectionTime)
+	result := boolValue(c.host.CheckedInAt.String() == "" || time.Since(time.Time(c.host.CheckedInAt)) <= MaxHostDisconnectionTime)
+	v.log.Debugf("%s validation result for host %s is: %s", getFunctionName(), c.host.ID.String(), result)
```

Comment:
once host is updated you will see `updated with ...` log, this log include validations info as well. so it will summarize the result, doesn't it help?

Reply:
does it always happen? U mean, for every validation run?
where is this print located in the code?

Reply:
I was thinking that it will happen here 
https://github.com/openshift/assisted-service/blob/0e701017abfc43eb08b4e45a2dbb8613f76c1f43/internal/host/host.go#L391
but it doesn't https://github.com/openshift/assisted-service/blob/0e701017abfc43eb08b4e45a2dbb8613f76c1f43/internal/host/hostutil/update_host.go#L85 
Maybe instead of updating it every time it should just report changes like `reportValidationStatusChanged`?

Reply:
I see that reportValidationStatusChanged() is being called only if didValidationChanged() returns True, and I did add a couple of prints to reportValidationStatusChanged(), won't that cover all prints for validation changes?

The other portion of this PR is returning validation results per each host, and each run will give a good indication if some validation keeps failing - it is possibly spammy, so this is why it is set for debug level.

Reply:
Then maybe log all the results once? otherwise you will have a line per validation per host, think about how will the log look like even if you have 5 hosts.
Maybe a single line after https://github.com/openshift/assisted-service/blob/master/internal/host/host.go#L377 ?